pkg/controllers/awsloadbalancercontroller/deployment.go

```diff
@@ -200,6 +224,30 @@ func desiredDeployment(name, namespace, image, vpcID, clusterName, awsRegion, cr
 	if controller.Spec.Config != nil && controller.Spec.Config.Replicas != 0 {
 		d.Spec.Replicas = pointer.Int32(controller.Spec.Config.Replicas)
 	}
+	if trustedCAConfigMapName != "" {
```

Comment:
Why don't you check for trustedCAVolumeName too?

Reply:
The volume is added by the operator, the user doesn't have a means to influence the volume. Also the same configmap [is used by the operator](https://github.com/openshift/aws-load-balancer-operator/blob/main/docs/proxy.md#running-operator) to talk to AWS at the staartup, so the operator would fail to start before we reach this point in code.